site/utils/apiUtils/index.ts

```diff
@@ -231,3 +231,16 @@ export const handleUpsertConsequence = async (
         throw e;
     }
 };
+
+export const formatAddOrEditUserBody = (body: object) => {
+    const operatorOrg = Object.entries(body)
+        .filter((item) => item.includes("operatorOrg"))
+        .flat();
+
+    const cleansedBody = Object.fromEntries(Object.entries(body).filter((item) => !item[0].startsWith("operatorOrg")));
+
+    return {
+        ...cleansedBody,
+        operatorOrg: operatorOrg[1] ? (JSON.parse(operatorOrg[1] as string) as object) : undefined,
```

Comment:
Do we need the casting here, `JSON.parse(operatorOrg[1])` seems to work

Reply:
I think so, as it throws typescript errors if I don't cast